### PR TITLE
Complete some TODOs

### PR DESCRIPTION
**Pull request by ayntee**

### CODE REVIEW

File: src/helpers/guilds/get_widget.ts [hunk 1]
```diff
@@ -11,6 +12,8 @@ export async function getWidget(guildId: string, options?: { force: boolean }) {
     if (!guild?.widgetEnabled) throw new Error(Errors.GUILD_WIDGET_NOT_ENABLED);
   }
 
-  // TODO: add return type
-  return await rest.runMethod("get", `${endpoints.GUILD_WIDGET(guildId)}.json`);
+  return await rest.runMethod<DiscordGuildWidget>(
```
Comment (itohatweb) [inline src/helpers/guilds/get_widget.ts:15]: Wrong type. There currently is no type for what discord returns

**Review by itohatweb — CHANGES_REQUESTED** [review]

**Review by Skillz4Killz — APPROVED** [review]

File: src/helpers/guilds/get_widget.ts (hunk 1 above)
Comment (ayntee) [inline src/helpers/guilds/get_widget.ts:15]:
```suggestion
  return await rest.runMethod(
```

File: src/helpers/guilds/get_widget.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { cacheHandlers } from "../../cache.ts";
 import { rest } from "../../rest/rest.ts";
+import { DiscordGuildWidget } from "../../types/guilds/guild_widget.ts";
```
Comment (ayntee) [inline src/helpers/guilds/get_widget.ts:3]:
```suggestion
```

**Review by itohatweb — APPROVED** [review]
LGTM

**Pull request merged**